### PR TITLE
Project are shown as card designs on the "home page" 

### PR DESCRIPTION
**Pull request by Steffengreiner**
**What was changed** 

Redesign of the project list style grid into individual cards for each project, showing additional information such as: 
1.) The measurement types associated with the project
2.) The collaborators associated with each project
3.) The project responsible party if present and the project manager

**Missing features**
User Avatars are not colorized, since we currently don't store the avatars for each user. Therefore this feature was delayed after discussion with SP until the user profile page is ready

**Technical issues**
1.) Due to how the information is loaded within the grid, only the information within the projectpreview bean can be filtered via the searchfield. (ProjectCode, Modification Date, Project Name)
This has been documented as a seperate issue and should be tackled seperately, 
since we encounter a similiar issue([DM-1230](https://qbicsoftware.atlassian.net/browse/DM-1230)) in the measurement and raw data grid. 

2.) The project list will show a deleted user associated with a project as anonymous instead of throwing an exception 
This can happen if there is a mismatch between the entries defined in the ACL list and the user table (E.g. by deleting the user manually in the database) 

[DM-1230]: https://qbicsoftware.atlassian.net/browse/DM-1230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementLookupService.java [hunk 1]
```diff
@@ -68,6 +68,14 @@ public List<NGSMeasurement> queryNGSMeasurementsBySampleIds(String termFilter,
     return new ArrayList<>(termList);
   }
 
+  public long countNGSMeasurementsBySampleIds(Collection<SampleId> sampleIds) {
```
Comment (wow-such-code) [inline project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementLookupService.java:71]: This seems not to be used by the project view, instead each experiment is used to count sample ids - might this be faster? also: why was it changed in this PR?

Comment (wow-such-code) [reply project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementLookupService.java:71]: ah, nevermind

File: user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java [hunk 2]
```diff
@@ -158,7 +158,11 @@ private static List<Entry> extractContactInfo(Project project) {
     var projectResponsible = new Div();
     project.getResponsiblePerson()
         .ifPresentOrElse(person -> projectResponsible.add(generateContactContainer(person)),
-            () -> projectResponsible.add(createNoPersonAssignedSpan()));
+            () -> {
+              Span noProjectResponsibleSelected = new Span("No Project Responsible Selected");
```
Comment (wow-such-code) [inline user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java:162]: nice

File: user-interface/src/main/java/life/qbic/datamanager/views/general/Tag.java [hunk 3]
```diff
@@ -6,11 +6,41 @@ public class Tag extends Span {
...
   }
 
   public Tag(String text, String toolTip) {
     this(text);
     setTitle(toolTip);
   }
+
+  public void setTagColor(TagColor tagColor) {
+    setClassName("tag");
+    addClassName(tagColor.getColor());
```
Comment (wow-such-code) [inline user-interface/src/main/java/life/qbic/datamanager/views/general/Tag.java:20]: "tag" class should already be set in the constructor?

**Review by wow-such-code — APPROVED** [review]
Looks good to me! Only a NIT comment.

File: user-interface/src/main/java/life/qbic/datamanager/views/general/Tag.java (hunk 3 above)
Comment (Steffengreiner) [reply user-interface/src/main/java/life/qbic/datamanager/views/general/Tag.java:20]: This is a sneaky trick on my part. `setclassName` replaces all current css classes set within the tag with the basic "tag" class. This allows us to remove any previously set color for this tag :)

**Pull request merged**